source/_docs/pantheon-workflow.md

```diff
@@ -12,13 +12,17 @@ Every Pantheon site comes with three environments: Dev, Test, and Live. Separate
   <p>Optimize your dev team and streamline internal workflows. Pantheon delivers custom workshops to help development teams master our platform and improve their internal DevOps.</p>
 </div>
 
-## Code Moves Up, Content Moves Down
-<img src="/source/docs/assets/images/workflow.png" alt="Dev Test and Live icon" style="border:0;margin-left:auto;margin-right:auto;display:block;">
+## Code VS Content
+
+One of the core concepts at the heart of the Pantheon workflow is the distinction between **code** and **files**.
 
-The core of the Pantheon workflow is to move code up from Dev to Test to Live and content down from Live to Test to Dev.
+ - **Code** is the core of your CMS, plugins/modules, themes, css, javascript - anything that builds and shapes your site.
+ - **Content** is your files (static images, posts, PDF files), and the database itself.
+
+### Code Moves Up, Content Moves Down
+<img src="/source/docs/assets/images/workflow.png" alt="Dev Test and Live icon" style="border:0;margin-left:auto;margin-right:auto;display:block;">
 
-- **Code** includes plugins, modules, themes, CSS, JS—anything that's under Git version control.
-- **Content** includes files not under Git version control, like images, PDFs, and the database.
+The main process of the Pantheon workflow is to move code up from Dev to Test to Live and content down from Live to Test to Dev. To facilitate this, we put [files](/docs/files/) into our distributed filesystem, Valhalla, and [code](/docs/code/) on to the application servers. When you build or migrate your site to Pantheon, configuring the correct paths initially will avoid complications down the road.
```

Comment:
Suggest moving the definitions for code and content back to their original placement, after this paragraph. So we present the reader with the concept before breaking it down?